### PR TITLE
update: consolidate power on/off docs

### PR DESCRIPTION
**Pull request by ArthurFlag**
Consolidate power off/on and service deletion docs.

### CODE REVIEW

File: docs/platform/concepts/service-power-cycle.md [hunk 1]
```diff
@@ -1,83 +1,153 @@
 ---
-title: Service power cycle
+title: Power a service on/off
```
Comment (wojcik-dorota) [inline docs/platform/concepts/service-power-cycle.md:2]:
```suggestion
title: Power on/off a service
```

File: docs/platform/concepts/service-power-cycle.md [hunk 2]
```diff
@@ -1,83 +1,153 @@
...
+import TabItem from '@theme/TabItem';
+import StaticIp from "@site/static/includes/static-ip-cost-warning.md";
+import AutoDelete from "@site/static/includes/auto-delete-poweredoff.md";
 
-:::warning
-Depending on service type and plan, data loss may happen during a
-service power off, so it is important for users to understand the
-consequences before powering off a service.
-:::
+Controlling when to power your Aiven services on and off allows you to control the use of resources and save credits.
```
Comment (wojcik-dorota) [inline docs/platform/concepts/service-power-cycle.md:12]:
```suggestion
Controlling when to power on and off your Aiven services allows you to control the use of resources and save credits.
```

File: docs/platform/concepts/service-power-cycle.md [hunk 3]
```diff
@@ -1,83 +1,153 @@
...
+Idle resources are released and only the necessary data is kept after a power off.
```
Comment (wojcik-dorota) [inline docs/platform/concepts/service-power-cycle.md:13]:
```suggestion
Idle resources are released and only the necessary data is kept after a power-off.
```

Comment (wojcik-dorota) [reply docs/platform/concepts/service-power-cycle.md:13]: I suggest using hyphens for nouns `power-on` and `power-off` in the whole doc.

File: docs/platform/concepts/service-power-cycle.md [hunk 4]
```diff
@@ -1,83 +1,153 @@
...
+  See [Backups at Aiven][backup].
+
+:::important
+<AutoDelete/>
 :::
 
-## Power on
+<StaticIp/>
+
+When a service is powered off, you can [delete it]
```
Comment (wojcik-dorota) [inline docs/platform/concepts/service-power-cycle.md:66]: Why the brackets? Also the full stop is missing.

File: docs/products/cassandra/howto.md [hunk 5]
```diff
@@ -32,7 +32,7 @@ Cluster management
 -   [Schedule automatic maintenance updates](/docs/platform/howto/prepare-for-high-load)
 -   [Upgrade a managed Aiven for Apache Cassandra service](/docs/platform/howto/scale-services)
 -   [Tag a managed Aiven for Apache Cassandra service](/docs/platform/howto/tag-resources)
--   [Power-off and delete a managed Aiven for Apache Cassandra service](/docs/platform/howto/pause-from-cli)
+-   [Power-off and delete a managed Aiven for Apache Cassandra service](/docs/platform/concepts/service-power-cycle)
```
Comment (wojcik-dorota) [inline docs/products/cassandra/howto.md:35]:
```suggestion
-   [Power off and delete a managed Aiven for Apache Cassandra service](/docs/platform/concepts/service-power-cycle)
```

File: docs/products/clickhouse/howto.md [hunk 6]
```diff
@@ -35,7 +35,7 @@ Cluster management
 -   [Schedule automatic maintenance updates](/docs/platform/howto/prepare-for-high-load)
 -   [Upgrade a managed Aiven for ClickHouse® service](/docs/platform/howto/scale-services)
 -   [Tag a managed Aiven for ClickHouse® service](/docs/platform/howto/tag-resources)
--   [Power-off and delete a managed Aiven for ClickHouse® service](/docs/platform/howto/pause-from-cli)
+-   [Power-off and delete a managed Aiven for ClickHouse® service](/docs/platform/concepts/service-power-cycle)
```
Comment (wojcik-dorota) [inline docs/products/clickhouse/howto.md:38]:
```suggestion
-   [Power off and delete a managed Aiven for ClickHouse® service](/docs/platform/concepts/service-power-cycle)
```

File: docs/products/mysql/howto.md [hunk 7]
```diff
@@ -55,7 +55,7 @@ Cluster management
 -   [Schedule automatic maintenance updates](/docs/platform/howto/prepare-for-high-load)
 -   [Upgrade a managed Aiven for MySQL® service](/docs/platform/howto/scale-services)
 -   [Tag a managed Aiven for MySQL® service](/docs/platform/howto/tag-resources)
--   [Power-off and delete a managed Aiven for MySQL® service](/docs/platform/howto/pause-from-cli)
+-   [Power-off and delete a managed Aiven for MySQL® service](/docs/platform/concepts/service-power-cycle)
```
Comment (wojcik-dorota) [inline docs/products/mysql/howto.md:58]:
```suggestion
-   [Power off and delete a managed Aiven for MySQL® service](/docs/platform/concepts/service-power-cycle)
```

File: docs/products/postgresql/howto.md [hunk 8]
```diff
@@ -102,7 +102,7 @@ Cluster management
 -   [Schedule automatic maintenance updates](/docs/platform/howto/prepare-for-high-load)
 -   [Upgrade a managed Aiven for PostgreSQL® service](/docs/platform/howto/scale-services)
 -   [Tag a managed Aiven for PostgreSQL® service](/docs/platform/howto/tag-resources)
--   [Power-off and delete a managed Aiven for PostgreSQL® service](/docs/platform/howto/pause-from-cli)
+-   [Power-off and delete a managed Aiven for PostgreSQL® service](/docs/platform/concepts/service-power-cycle)
```
Comment (wojcik-dorota) [inline docs/products/postgresql/howto.md:105]:
```suggestion
-   [Power off and delete a managed Aiven for PostgreSQL® service](/docs/platform/concepts/service-power-cycle)
```

**Review by wojcik-dorota — CHANGES_REQUESTED** [review]
Mostly minor editorial stuff :-)

File: docs/platform/concepts/service-power-cycle.md (hunk 4 above)
Comment (ArthurFlag) [reply docs/platform/concepts/service-power-cycle.md:66]: I forgot the link 🙃 thanks

**Review by wojcik-dorota — APPROVED** [review]

**Pull request merged**